src/ToDoRequestService.js

```diff
@@ -13,6 +13,9 @@ const ToDoRequestService = {
   deleteToDo(id) {
     return RestClient.delete('/todos/' + id);
   },
+  editToDo(newToDo) {
+    return RestClient.patch('/todos/', { name: newToDo });
+  },
```

Comment:
You need to include the id in the url when performing a patch. Also why are you creating a new object with a name field equal to the newToDo? 'newToDo' should be a ToDo object with a name field.